### PR TITLE
Fix: do not merge current branch

### PR DESCRIPTION
**Pull request by randy3k**
`get_selected_branch` returns a tuple `(remote, branch_name)`, we need to compare the second element.

### CODE REVIEW

**Comment by asfaltboy** [comment]
LGTM

File: core/interfaces/branch.py [hunk 1]
```diff
@@ -513,7 +513,7 @@ def get_selected_branches(self):
         branches = set()
         for sel in self.view.sel():
             for name in self.get_selected_branch(sel):
-                if name and not name == current_branch_name:
+                if name and not name[1] == current_branch_name:
```
Comment (stoivo) [inline core/interfaces/branch.py]: Should we check that name[0] is None since I might want to merge remotea/master in to master

**Comment by asfaltboy** [comment]
Oh! @stoivo good catch! Can you submit a quick PR for this?

**Comment by stoivo** [comment]
I will

**Pull request merged**